app/routes.py

```diff
@@ -67,6 +67,9 @@ def posts():
 @login_required # allows only a logged in user to access account page
 def feed():
     job_posts = Post.query.all()
+    form = FeedApplyForm()
+    if form.validate_on_submit():
+        application = Application(user_id = current_user.id, post_id = )
```

Comment:
do the parameters in the function "Application" on line 72 all need to have a value assigned to them? will writing just "post_id =" lead to a syntax error?

Reply:
I believe the issue is that you're viewing an outdated commit from when the routes.py file was not fully complete. In the final routes.py file, line 72 is:
application = Application(user_id = current_user.id, cover_letter = form.cover_letter.data, post_id = form.post_id.data)